src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/appointments/AppointmentService.kt

```diff
@@ -65,7 +65,7 @@ class AppointmentService(
         endTime = dto.endTime?.let { LocalDateTime.of(dto.eventDate, it) }
         eventDate = dto.eventDate
         eventSubType = subType
-        comment = dto.comment
+        comment = formatComment(eventSubType, dto.comment)
```

Comment:
This the main change - apply the formatted comment to updates (already been done for create).